test/controllers/omdb_import_controller_test.rb

```diff
@@ -1,7 +1,19 @@
 require "test_helper"
 
 class OMDBImportControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  test "can add movie" do
+    assert_difference("Movie.count") do
+      VCR.use_cassette("post_imdb_id") do
+        post "/omdb_import",
```

Comment:
Når params ikke er mere kompliceret så skriver vi det som regel på 1 linje for at gøre det lidt overskueligt.
```ruby
post "/omdb_import", params: {imdb_id: ""}
```